net/nut/Makefile

```diff
@@ -9,28 +9,22 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nut
 PKG_VERSION:=2.7.4
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.networkupstools.org/source/2.7/
 PKG_MD5SUM:=3ba53656933d7471f95140b32a5b8d5c
-PKG_MAINTAINER:=Martin Rowe <martin.p.rowe@gmail.com>
+PKG_MAINTAINER:=Daniel Dickinson <lede@daniel.thecshore.com>
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=LICENSE-GPL2
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/nut-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_INSTALL:=1
 
 PKG_CONFIG_DEPENDS:= \
-	CONFIG_NUT_CLIENTS_UPSC \
-	CONFIG_NUT_CLIENTS_UPSCMD \
-	CONFIG_NUT_CLIENTS_UPSLOG \
-	CONFIG_NUT_CLIENTS_UPSMON \
-	CONFIG_NUT_CLIENTS_UPSRW \
-	CONFIG_NUT_CLIENTS_UPSSCHED \
 	CONFIG_NUT_DRIVER_SNMP \
 	CONFIG_NUT_DRIVER_USB \
-	CONFIG_NUT_SERVER \
+	CONFIG_NUT_DRIVER_SERAL \
```

Comment:
@dfc-d I suspect CONFIG_NUT_DRIVER_SERAL should be CONFIG_NUT_DRIVER_SERIAL.
typo?
Thanks for all your work on this by the way.